serving/samples/jib-app-java/README.md

```diff
@@ -0,0 +1,99 @@
+# Jib Sample App
+
+A sample app that demonstrates using [Jib](https://github.com/GoogleContainerTools/jib) to build Java applications
+and running with Knative Serving.
+
+Jib can package any Java application without using the Dockerfile.
+
+This deploys the [Helloworld Java](https://github.com/knative/docs/tree/master/serving/samples/helloworld-java)
```

Comment:
Does it make sense to build an example from some other project, e.g. a sample jetty webapp or the like?

Reply:
this is what the dotnet app example has done

Reply:
The [`buildpack-app-dotnet`](https://github.com/knative/docs/tree/master/serving/samples/buildpack-app-dotnet) deploys a sample from `cloudfoundry-samples` which doesn't have a `Dockerfile` in the same directory.

Something like https://github.com/cloudfoundry-samples/cf-sample-app-spring would be a good example of this working with existing source code rather than a special configuration.

Reply:
agreed if we can use cncf buildpack container; i haven't found it...